### ISSUE TITLE
Global TOC missing

### ISSUE THREAD

**Issue by StefanUlbrich** [bashtage/sphinx-material#72]
Hi, thanks for this amazing theme!

However, for me the global TOC does not appear on the right. I use sphinx version 3.2.1. The console shows a few errors but I don't think they explain the missing TOC

```
Access to XMLHttpRequest at 'file:///Users/stef/bodyschema/bs-pose-fit/doc/build/html/versions.json' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https.
versions.json:1 Failed to load resource: net::ERR_FAILED
application.js:244 Uncaught TypeError: Cannot read property 'style' of undefined
    at e.t.reset (application.js:244)
    at e.t.unlisten (application.js:74)
    at MediaQueryList.MatchMedia.handler_ (application.js:79)
t.reset @ application.js:244
t.unlisten @ application.js:74
MatchMedia.handler_ @ application.js:79
application.js:244 Uncaught TypeError: Cannot read property 'style' of undefined
    at e.t.reset (application.js:244)
    at e.t.unlisten (application.js:74)
    at MediaQueryList.MatchMedia.handler_ (application.js:79)
```

**Comment by bashtage** [comment]
You need to run a local webserver to avoid this type of issue with chrome.

**Comment by bashtage** [comment]
Can you build the demo site? Does it look right?

**Comment by StefanUlbrich** [comment]
Sorry for the accidental duplicate of #73
